src/main/java/org/team340/robot/Constants.java

```diff
@@ -35,5 +35,8 @@ public static final class RobotMap {
         public static final int kBrEncoder = 13;
 
         public static final int kCanandgyro = 14;
+
+        public static final int kIntakeMotor = 20;
```

Comment:
```suggestion
        public static final int kIntakeMotor = 40;
```
The elevator subsystem is utilizing the 2x CAN ID range, so we should probably bump the intake up to the 4x range. For posterity, here's what we are looking at for subsystem CAN ranges:

Drivetrain: 2-9, 1x
Elevator: 2x
Gooseneck: 3x
Intake: 4x
Climber: 5x
Potential Algae Mech: 6x